COLLADAMaya/include/COLLADAMayaLightProbeExporter.h

```diff
@@ -0,0 +1,188 @@
+/*
+    Copyright (c) 2008-2009 NetAllied Systems GmbH
```

Comment:
please fix copyright

Copyright (c) 2015 Starbreeze AB

are some portions of the code below from Feeling Software ? Sony? Alias?